test/fixtures/test.html

```diff
@@ -0,0 +1 @@
+<div class="test">test</div>
```

Comment:
maybe `literal-imported-file.html` as filename? likewise for multiroot. `test.html` is hard to grok in this big list of fixtures